osquery/tables/applications/posix/docker.cpp

```diff
@@ -578,37 +591,40 @@ QueryData genContainerProcesses(QueryContext& context) {
         std::vector<std::string> vector;
         BOOST_FOREACH (const auto& v, processes.second) {
           vector.push_back(v.second.data());
-        }
-
-        // On OS X, ps_args are not being honored. Check number of columns
-        if (vector.size() != 21) {
```

Comment:
This is an important check otherwise we might try to read beyond the vector length.